### ISSUE TITLE
failed to run demo

### ISSUE THREAD

**Issue by jainy** [parklab/xTea#4]
Hi, 

Thank you for  xTea.

I am interested in using the xTea for identifying TEs in my data. I installed the tool and I was testing with the script in demo folder.

I changed the paths of the corresponding files in run_gnrt_pipeline.sh and ran it using 'sh run_gnrt_pipeline.sh'. Rest is default parameters.

But it gave me the following error.   Can you please help me with that. 

Please let me know if you need any other info

Thank you
Jainy

sh run_gnrt_pipeline.sh
Traceback (most recent call last):
  File "/home/jainy/software/xTea/xtea/gnrt_pipeline_local.py", line 1170, in <module>
    smemory, ncores, sf_sbatch_sh, "null", b_lsf, b_slurm, b_resume)
  File "/home/jainy/software/xTea/xtea/gnrt_pipeline_local.py", line 756, in gnrt_running_shell
    split_bam_list(m_id, sf_bams, sf_10X_bams, l_rep_type, s_wfolder, sf_case_control_bam_list)
  File "/home/jainy/software/xTea/xtea/gnrt_pipeline_local.py", line 840, in split_bam_list
    sid = fields[0]
IndexError: list index out of range

**Comment by simoncchu** [comment]
Could you paste the content of the `sample_bams.txt` file , and also the content of the exact command?

**Comment by jainy** [comment]
Thank you for your quick response!

Content of sample_bams.txt file
1-00866      /ybod2/kssell/U_BAMS/PolishedBams/1-00866.cram

Content of run_gnrt_pipeline.sh
#!/bin/bash

SAMPLE_ID=/ybod2/jainy/Project3/xTea-samplerun/sample_id.txt
BAMS=/ybod2/jainy/Project3/xTea-samplerun/illumina_bam_list.txt
X10_BAM=null
WFOLDER=/ybod2/jainy/Project3/xTea-samplerun/outputxTea_sample/
OUT_SCRTP=submit_jobs.sh
TIME=0-05:00
REP_LIB=/home/jainy/software/xTea/rep_lib_annotation/
REF=/kbod2/ReferenceGenomes/GRCh37/human_g1k_v37_decoy_phix.fasta
GENE=/home/jainy/software/xTea/gene_annotation/gencode.v28lift37.annotation.gff3
XTEA=/home/jainy/software/xTea/xtea/
BLK_LIST=/home/jainy/software/xTea/rep_lib_annotation/rep_lib_annotation/blacklist/hg19/centromere.bed

#
python ${XTEA}"gnrt_pipeline_local.py" -i ${SAMPLE_ID} -b ${BAMS} -x ${X10_BAM} -p ${WFOLDER} -o ${OUT_SCRTP} -q short -n 8 -m 16 -t ${TIME} \
-l ${REP_LIB} -r ${REF} -g ${GENE} --xtea ${XTEA}  --nclip 4 --cr 2 --nd 5 --nfclip 4 --nfdisc 5 --flklen 3000 -f 5907  -y 7  --blacklist ${BLK_LIST}


Please let me know if you need any other info.

Thank you
Jainy

**Comment by simoncchu** [comment]
@jainy The file and content looks good for me. Could you double check there is no empty lines in the file?

**Comment by jainy** [comment]
Hi ,

Thank you! I checked there was an empty line. I removed it and ran sh run_gnrt_pipeline.sh command. It created the submit_jobs.sh script. I ran it using sh submit_jobs.sh command.

This is what I got:

submit_jobs.sh: line 3: sbatch: command not found
submit_jobs.sh: line 4: sbatch: command not found
submit_jobs.sh: line 5: sbatch: command not found

Since I am running the job in a single server, how should I do it.

Thank you
Jainy
